src/Elasticsearch/Endpoints/Ingest/Pipeline/Get.php

```diff
@@ -23,10 +23,9 @@ class Get extends AbstractEndpoint
     public function getURI()
     {
         if (isset($this->id) !== true) {
-            throw new Exceptions\RuntimeException(
```

Comment:
Why the removal?  ID is required for the GetPipeline API: https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/ingest.get_pipeline.json#L12